subsys/bluetooth/host/conn.c

```diff
@@ -614,429 +592,460 @@ static inline uint16_t conn_mtu(struct bt_conn *conn)
 #endif /* CONFIG_BT_CONN */
 }
 
-static int do_send_frag(struct bt_conn *conn, struct net_buf *buf, uint8_t flags)
+static bool is_classic_conn(struct bt_conn *conn)
 {
-	struct bt_conn_tx *tx = tx_data(buf)->tx;
-	uint32_t *pending_no_cb = NULL;
-	unsigned int key;
-	int err = 0;
+	return (IS_ENABLED(CONFIG_BT_CLASSIC) &&
+		conn->type == BT_CONN_TYPE_BR);
+}
 
-	LOG_DBG("conn %p buf %p len %u flags 0x%02x", conn, buf, buf->len,
-		flags);
+static bool is_iso_conn(struct bt_conn *conn)
+{
+	return IS_ENABLED(CONFIG_BT_ISO_TX) &&
+		conn->type == BT_CONN_TYPE_ISO;
+}
```

Comment:
As mentioned in the previous, this gives a mismatch between the function name and the implementation given that it uses `CONFIG_BT_ISO_TX` instead of `CONFIG_BT_ISO`. 

It's OK to keep as is, as it is a small optimization, but in that case I would suggest to add a comment stating why we can use `CONFIG_BT_ISO_TX`. 

If we want to keep the `CONFIG_BT_ISO_TX`, then we should also be able to use `CONFIG_BT_CONN_TX` for `is_le_conn`, right?

Reply:
yes I'm sorry I had it renamed but forgot to push :/
I'll rename to `is_iso_tx_conn()`. Agreed for `is_le_conn` I can also rename it.

Reply:
Changed my mind for `is_le_conn()`: We can't have RX-only ACL links currently so I don't see the point of using `CONFIG_BT_CONN_TX` for le/br/acl too.